mkosi/__init__.py

```diff
@@ -4387,6 +4387,9 @@ def finalize_default_tools(config: Config, *, resources: Path) -> Config:
         "--incremental", str(config.incremental),
         *([f"--package={package}" for package in config.tools_tree_packages]),
         *([f"--package-directory={os.fspath(directory)}" for directory in config.tools_tree_package_directories]),  # noqa: E501
+        *([f"--build-sources={tree}" for tree in config.build_sources]),
```

Comment:
The build sources here are needed to have them be usable by the prepare script, right?

Reply:
Yeah